lms/envs/common.py

```diff
@@ -515,6 +515,17 @@
         'OPTIONS': {
             'name': 'tracking'
         }
+    },
+    'oli': {
+        'ENGINE': 'track.backends.oli.OLIAnalyticsBackend',
+        'OPTIONS': {
+            'oli_analytics_service_url': 'https://cb-jorr-4.appspot.com/oli_target/rest/modules/learning_analytics/',
+            'path': 'event',
+            'key': 'learning-analytics',
+            'secret': 'correct horse battery staple',
+            'timeout_in_secs': 5.0,
+            'course_ids': ['edX/DemoX/Demo_Course']
```

Comment:
According to the documentation, yes admins would put the whitelisted courses here.